core/src/apps/stellar/layout.py

```diff
@@ -44,18 +50,28 @@ async def require_confirm_init(
         )
 
 
-async def require_confirm_timebounds(ctx, start: int, end: int):
-    await confirm_timebounds_stellar(ctx, start, end)
+async def require_confirm_timebounds(ctx: Context, start: int, end: int) -> None:
+    await confirm_properties(
+        ctx,
+        "confirm_timebounds",
+        title="Confirm timebounds",
+        props=(
+            ("Valid from (UTC)", str(start) if start > 0 else "[no restriction]"),
+            ("Valid to (UTC)", str(end) if end > 0 else "[no restriction]"),
```

Comment:
I know it's a change wrt to original code, but shouldn't we show human readable time here like T1 does?

Reply:
we can't currently because we don't have epoch-to-datetime decoding.

and we will get it with uPy 1.14 (see #1535) so this will be fixed then